libs/payments/ui/src/lib/client/components/CouponForm/index.tsx

```diff
@@ -0,0 +1,118 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
One or both of the `CouponForm` components should be renamed to more appropriately match their intended purpose.